core/src/it/scala/org/ensime/fixture/EnsimeConfigFixture.scala

```diff
@@ -52,6 +52,8 @@ object EnsimeConfigFixture {
     )
     System.err.flush()
     sys.exit(1)
+  } else {
+    println(dotEnsime.getAbsolutePath)
```

Comment:
I'll remove the comment on TypeCheckModule(moduleName). Did I leave out more comments?